pkg/middleware/loggermw/logger.go

```diff
@@ -170,8 +198,10 @@ func SanitizeURL(s string) (string, error) {
 
 	// strip out sensitive query strings
 	values := u.Query()
-	for _, query := range sensitiveQueryStrings {
-		values.Del(query)
+	for _, check := range sensitiveQueryChecks {
+		if check.checker(values) {
```

Comment:
I wonder should this check be case insensitive

Reply:
Yes I think that would make sense

Reply:
So the only way I can think of supporting case insensitive query params is to loop all the provided query params and then make them lower case to compare with the sensitive dictionary (which should be lower case).

But then I need to loop over all query strings every request. Is that ok?

Reply:
> But then I need to loop over all query strings every request. Is that ok?

I think that should be okay